static/sass/_pattern_cve.scss

```diff
@@ -17,7 +17,7 @@
   }
 
   .cve-table {
-    font-size: 0.8rem;
+    font-size: 0.875rem;
```

Comment:
Ideally this should refer to the variable so if we change something it gets updated here automatically, e.g.:
```
    font-size: #{map-get($font-sizes, small)}rem;
```

Small text also has smaller line-height, and some other adjusted properties, so something like this may be better:

```
@extend %small-text;
```